server-spi-private/src/main/java/org/keycloak/broker/provider/IdentityProvider.java

```diff
@@ -132,4 +134,9 @@ interface AuthenticationCallback {
      */
     IdentityProviderDataMarshaller getMarshaller();
 
+    /**
+     * Checks whether a mapper is supported for this Identity Provider.
+     */
+    boolean isMapperSupported(IdentityProviderMapper mapper);
```

Comment:
Wouldn't make more sense to use a default method here?

Reply:
Hi @pedroigor! I took over since @danielFesenmeyer is on PTO. I included your feedback.

Reply:
Thanks, it should also make it easier to not break others' implementation while keeping the default behavior.